glide/src/main/java/dev/chrisbanes/accompanist/glide/GlideImage.kt

```diff
@@ -46,6 +47,22 @@ import dev.chrisbanes.accompanist.imageloading.toPainter
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.suspendCancellableCoroutine
 
+/**
+ * Ambient containing the preferred [RequestManager] to use in [GlideImage].
+ */
+val AmbientRequestManager = staticAmbientOf<RequestManager?> { null }
```

Comment:
Should this be `AmbientGlideRequestManager` if it's only used in Glide? Otherwise, it can be confusing when listing all the possible Ambients using auto-complete

Reply:
You shouldn't be using any more than one of these libraries 😄